src/mscorlib/src/System/String.cs

```diff
@@ -174,18 +174,28 @@ unsafe public char[] ToCharArray(int startIndex, int length)
             return Array.Empty<char>();
         }
 
-        public static bool IsNullOrEmpty(String value)
+        public static bool IsNullOrEmpty(string value)
         {
-            return (value == null || value.Length == 0);
+            return (value is null || value.Length == 0);
         }
 
-        public static bool IsNullOrWhiteSpace(String value)
+        public static bool IsNullOrWhiteSpace(string value)
         {
-            if (value == null) return true;
+            if (IsNullOrEmpty(value)) return true;
+
+            return IsWhiteSpace(value);
```

Comment:
Isn't this just:
return IsNullOrEmpty(value) || IsWhitespace(value);
?

Reply:
Yes :)